src/components/molecules/NavItem/index.tsx

```diff
@@ -0,0 +1,47 @@
+import React from 'react'
+import { Box, Typography } from '@mui/material'
+import theme from '../../../theme/theme'
+import Icon from '../../Atoms/Icon'
+
+interface InputProps {
+  iconName: string
+  text: string
+  onNavItemClick: (value: string) => void
+  active: boolean
+}
+const NavItem = ({ iconName, text, onNavItemClick, active }: InputProps) => {
+  return (
+    <Box
+      onClick={() => {
+        onNavItemClick(iconName)
+      }}
+      display="flex"
+      sx={{
```

Comment:
Move all sx objects to top. 
Declare styles at top of the file after imports and use that variables here. 
example:
const styleMap = {
  checkBoxStyle: {
    color : theme.pallete.primary,
  },
  tableCellStyle: {
    display: 'flex',
    alignItems: 'center',
    justifyContent: 'center',
  }
}

<CheckBox sx={styleMap.checkBoxStyle} />
<TableCell sx={styleMap.tableCellStyle} />

Reply:
ok,done